### PR TITLE
Fix tentacles tests

### PR DESCRIPTION
**Pull request by Werdor**

### CODE REVIEW

File: backtesting/abstract_backtesting_test.py [hunk 1]
```diff
@@ -222,8 +222,8 @@ def _assert_results(self, run_results, profitability, bot):
             self._assert_results(run_results, profitability_1, bot)
 
     def _register_only_strategy(self, strategy_evaluator_class):
-        for evaluatotor_name in self.config[CONFIG_EVALUATOR]:
```
Comment (GuillaumeDSM) [inline backtesting/abstract_backtesting_test.py]: woops 👍

File: tests/test_utils/bot_management.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    event_loop.call_later(time, stop_bot, bot)
+
+
+async def start_bot_with_raise(bot, run_in_new_thread=False):
+    with pytest.raises(CancelledError):
+        await start_bot(bot, run_in_new_thread)
+
+
+async def start_bot(bot, run_in_new_thread=False):
+    await bot.start(run_in_new_thread)
```
Comment (GuillaumeDSM) [inline tests/test_utils/bot_management.py]: 👍

File: tools/commands.py [hunk 3]
```diff
@@ -51,7 +51,7 @@ def package_manager(config, commands, catch=False, force=False, default_git_bran
                 force = True
                 commands.pop(commands.index(TENTACLES_FORCE_CONFIRM_PARAM))
             tentacle_manager = TentacleManager(config)
-            tentacle_manager.parse_commands(commands, force=force, default_git_branch=default_git_branch)
```
Comment (GuillaumeDSM) [inline tools/commands.py]: Why removing this ? This means we can't specify the tentacle branch from octobot

Comment (Werdor) [reply tools/commands.py]: I shouldn't have commit this. I have a local problem with Tentacle-Manager package.

Comment (Werdor) [reply tools/commands.py]: Done 👍

**Pull request merged**